plugins.json

```diff
@@ -881,9 +881,9 @@
         "author": "Andrei Volgin, ipunkt Business Solutions",
         "license": "MIT",
         "description": "This Plugin is used to automatically alert users when they've lost internet connectivity.",
-        "homepage": "https://github.com/ipunkt/OfflineKanboardPlugin",
-        "readme": "https://github.com/ipunkt/OfflineKanboardPlugin/blob/master/README.md",
-        "download": "https://github.com/ipunkt/OfflineKanboardPlugin/releases/download/1.0.0/OfflineKanboardPlugin-1.0.0.zip",
+        "homepage": "https://github.com/ipunkt/KanboardOffline",
+        "readme": "https://github.com/ipunkt/KanboardOffline/blob/master/README.md",
+        "download": "https://github.com/ipunkt/KanboardOffline/releases/download/1.0.0/OfflineKanboardPlugin-1.0.0.zip",
```

Comment:
This link returns a 404:

```
wget https://github.com/ipunkt/KanboardOffline/releases/download/1.0.0/OfflineKanboardPlugin-1.0.0.zip
--2019-10-15 19:53:20--  https://github.com/ipunkt/KanboardOffline/releases/download/1.0.0/OfflineKanboardPlugin-1.0.0.zip
Resolving github.com (github.com)... 192.30.253.112
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-10-15 19:53:20 ERROR 404: Not Found.
```

Reply:
@fguillot Thank you, Done.
 Didn't make a release after renaming.